extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/process/ProcessConfigCreator.scala

```diff
@@ -32,4 +34,12 @@ trait ProcessConfigCreator extends Serializable {
   def classExtractionSettings(modelDependencies: ProcessObjectDependencies): ClassExtractionSettings =
     ClassExtractionSettings.Default
 
+  def modelSettings(modelDependencies: ProcessObjectDependencies): ModelSettings = {
+    Try(
```

Comment:
1. Catching all exceptions without any logging is never a good option :) Let's use Ficus `.getAs[Boolean]` which returns `Option[Boolean]` - see how it is done in `DesignerConfig`
2. Let's make configuration parsing logic eager; it should be a field of `ProcessObjectDependencies` instead of method. Eventually we should create it during creation of `ProcessingTypeConfig.modelConfig`